test/contrib/test_ssltransport.py

```diff
@@ -151,28 +151,39 @@ def test_unbuffered_text_makefile(self):
             response = consume_socket(ssock)
             validate_response(response)
 
-    @pytest.mark.skip(reason="Disabled as it's currently flaky.")
+    @pytest.mark.timeout(PER_TEST_TIMEOUT)
     def test_unwrap_existing_socket(self):
         """
         Validates we can break up the TLS layer
-        Request is sent over TLS. response received over regular TCP.
-
-        Currently disabled as its flaky on occassion.
+        A full request/response is sent over TLS, and later over plain text.
         """
 
         def shutdown_handler(listener):
             sock = listener.accept()[0]
             ssl_sock = self.server_context.wrap_socket(sock, server_side=True)
+
             request = consume_socket(ssl_sock)
             validate_request(request)
-            sock = ssl_sock.unwrap()
-            sock.send(sample_response())
+            ssl_sock.send(sample_response())
```

Comment:
nit: should these be `sendall()` instead of `send()`?

Reply:
Good catch, as you said, I don't think it's making much of a difference with such small buffers but it doesn't hurt to be thorough.